src/core/entities/crux-user.ts

```diff
@@ -30,6 +30,7 @@ export interface ICruxUserInformation {
 export interface ICruxUserData {
     configuration: ICruxUserConfiguration;
```

Comment:
can we put blacklistAddresses this inside ICruxUserConfiguration?
What is our policy on which kind of attributes go inside ICruxUserConfiguration?

Reply:
Added BlacklistedCruxUsers to CruxUser's configuration